packages/ag-charts-community/src/chart/axis/axis.ts

```diff
@@ -1085,8 +1085,12 @@ export abstract class Axis<S extends Scale<D, number, TickInterval<S>> = Scale<a
     }
 
     protected calculateDomain() {
-        const visibleSeries = this.boundSeries.filter((s) => this.includeInvisibleDomains || s.isEnabled());
-        const domains = visibleSeries.flatMap((series) => series.getDomain(this.direction));
+        const { includeInvisibleDomains, boundSeries, direction } = this;
+        const visibleSeries = boundSeries.filter((s) => includeInvisibleDomains || s.isEnabled());
```

Comment:
```suggestion
        const visibleSeries = this.includeInvisibleDomains ? boundSeries : boundSeries.filter((s) => s.isEnabled());
```
nit: I assume that you destructed `includeInvisibleDomains` to reduce lookup time, but if that's the case wouldn't it be simpler to skip filtering altogether?

Reply:
Yes - good point